active_model_serializers.gemspec

```
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.1'
 
-  rails_versions = ['>= 4.1', '< 7.1']
+  rails_versions = ['>= 4.1', '< 7.2']
```

Would it instead not be nicer to only define a "has to greater than" version - and mention explicitly in the README that it is only tested up to version 7.1?

```suggestion
  rails_versions = ['>= 4.1']
```

The gem paper_trail defines it like this in the gemspec:

` s.add_development_dependency "rails", ::PaperTrail::Compatibility::ACTIVERECORD_GTE`

And displays a clear message when using a new rails version: https://github.com/paper-trail-gem/paper_trail/blob/d84270fec4141f424006bdf4ad6a4cff4e1bd452/lib/paper_trail/compatibility.rb#L19

---

I remember we had some discussion on this in the past, but decided to keep it work like the current way

cc @bf4

---

Yeah, there was some talk of that in https://github.com/rails-api/active_model_serializers/pull/2428 which I based these changes on... if that helps.

---

As the gem ages, I'm ok with removing the hard upper limit just documenting the known compatibility,  in order to ease the maintenance burden

---

@schinery could you please make this change too. 

We can also do it later, in which case I'll keep in mind to make the modification :)

---

@wasifhossain done https://github.com/rails-api/active_model_serializers/pull/2453/commits/511901a22d7511f8439578b2d8360758958f7889